deploy/helm/pulumi-operator/crds/program-crd.yaml

```diff
@@ -0,0 +1,186 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Can these be copied from deploy/crds in a make target?

Reply:
I can add an cp to the `generate_crds.sh` file!